app/src/main/res/values/dimens.xml

```diff
@@ -14,58 +13,40 @@
   <dimen name="elevation_4dp">4dp</dimen>
   <dimen name="elevation_0dp">0dp</dimen>
   <dimen name="padding_296dp">296dp</dimen>
```

Comment:
let's update this to0.

Reply:
@ranjsa I think we are not using this padding anywhere, could you please verify this and remove if not needed?

Reply:
yes @anandwana001 we were not using this anywhere so I removed it